apidef/api_definitions.go

```diff
@@ -591,6 +527,75 @@ type RequestSigningMeta struct {
 	SignatureHeader string   `bson:"signature_header" json:"signature_header"`
 }
 
+// GraphQLConfig is the root config object for GraphQL middlewares
```

Comment:
GoDoc convention is to finish with `.`

Better to finish all GoDoc with `.` in this PR.

Reply:
Done.